src/components/Article.js

```diff
@@ -43,22 +48,32 @@ export const Article = ({ lang, title }) => {
     window.location.hash = `/quickfacts/${lang}/${title}`
   }
 
-  if (!article) {
-    return 'Loading...'
-  }
-
   const hasImage = !!article.imageUrl
 
+  const pagerEvent = {
+    nextPage: () => {
+      const sectionLength = article.sections.length
+      const nextSection = currentSection + 1
+
+      if (nextSection < sectionLength) setCurrentSection(nextSection)
```

Comment:
Please always use curly braces with conditions.

In this case, however, it can be simplified to
`setCurrentSection( nextSection < sectionLength ? nextSection : 0 )`

Reply:
see https://github.com/wikimedia/wikipedia-kaios/pull/15/commits/c823f06edc11a023b89ccda5cf7723fc06bad250